src/pymodaq_gui/parameter/utils.py

```diff
@@ -166,14 +166,33 @@ def iter_children(param, childlist=[]):
     return childlist
 
 
-def iter_children_params(param, childlist=[]):
+def iter_children_params(param, childlist=[], filter_type=[], filter_name=[], selectFilter=False):
     """Get a list of parameters under a given Parameter
+        | Returns all children parameters.
```

Comment:
could you reformat the whole docstring using numpy style (the one here is an old attempt to follow pyqtgraph style but it has been abandoned)